src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -184,8 +184,17 @@ def validate_jwt_svid(self, token: str, audience: str) -> JwtSvid:
         Returns:
             JwtSvid: If the token and audience could be validated.
         """
+        try:
```

Comment:
why not test the token and audience provided? at least if they are not None or empty before sending these to the agent? I know these validations are performed at the agent side but we shall be able to provide a quicker error if we avoid the agent call.